graphql/resolvers.go

```diff
@@ -3350,6 +3352,21 @@ func (r *taskResolver) BaseTask(ctx context.Context, obj *restModel.APITask) (*r
 	return apiTask, nil
 }
 
+func (r *taskResolver) TaskDuration(ctx context.Context, obj *restModel.APITask) (*restModel.APIDuration, error) {
+	// TaskDuration is sometimes added via aggregation when Task is resolved via GetTasksByVersion.
+	if obj.TaskDuration != 0 {
+		return &obj.TaskDuration, nil
+	}
+	// If status is running / dispatched, calculate the time elapsed since it became active.
```

Comment:
Activated doesn't mean started time, it means the time at which the user marked it to be scheduled. More accurate would be StartTime (not sure if this would be set for TaskDispatched so that may need to be 0; that makes sense to me though since that's Evergreen getting the task ready still, and not how long the task has been running for).